### PR TITLE
docs: update PR template

### PR DESCRIPTION
**Pull request by xinxingjiang1**
## Context

> Why are you making this change?

https://opendoor.atlassian.net/browse/INFRA-3031

## Origin

> What team/individual requested this change? Please link to a JIRA ticket or other documentation if applicable. If you identified this issue yourself please write "Me" (and still link to any additional context if it exists).

Me

## Summary of Changes

* Add a note to require `terraform plan` output or link in `Test Plan` section.

## Test Plan

No need.

## Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to external documentation if necessary (READMEs, Confluence, etc.)
- [ ] I have checked that other PRs this PR depends on have already been deployed.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 ## Test Plan
 
 <!-- How have you tested this change, and what further testing will be done? What’s the riskiest part of this PR? How will you test and monitor that? -->
+<!-- If you PR changes a Terraform file, include the terraform plan output or link before asking for code review. -->
```
Comment (markrmullan) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- If your PR changes a Terraform file, include the terraform plan output or link before asking for code review. -->
```

**Review by markrmullan — APPROVED** [review]
Overall this LGTM pending a typo fix.

As an alternative, it could be worth exploring whether this would be better applied at the `Dangerfile` level. A benefit being that you could add a comment to the PR iff the PR touched a `.tf` file, rather than adding this reminder globally, to all PRs.

https://github.com/opendoor-labs/code/blob/master/code/pipeline/danger/Dangerfile

**Review by xdu-opendoor — APPROVED** [review]

**Comment by xinxingjiang1** [comment]
> Overall this LGTM pending a typo fix.
> 
> As an alternative, it could be worth exploring whether this would be better applied at the `Dangerfile` level. A benefit being that you could add a comment to the PR iff the PR touched a `.tf` file, rather than adding this reminder globally, to all PRs.
> 
> https://github.com/opendoor-labs/code/blob/master/code/pipeline/danger/Dangerfile

Sorry, I just noticed your comment.

TF Cloud will solve this problem by automating "terraform plan" in every PR if it changes a .tf file. So this option or the other one is a short term fix. I prefer a simpler solution at this time.

**Comment by markrmullan** [comment]
> > Overall this LGTM pending a typo fix.
> > As an alternative, it could be worth exploring whether this would be better applied at the `Dangerfile` level. A benefit being that you could add a comment to the PR iff the PR touched a `.tf` file, rather than adding this reminder globally, to all PRs.
> > https://github.com/opendoor-labs/code/blob/master/code/pipeline/danger/Dangerfile
> 
> Sorry, I just noticed your comment.
> 
> TF Cloud will solve this problem by automating "terraform plan" in every PR if it changes a .tf file. So this option or the other one is a short term fix. I prefer a simpler solution at this time.

Cool! Sounds good to me 👍

**Pull request merged**